_pages/welcome-to-TTS/classes/olu.md

```diff
@@ -21,15 +21,14 @@ Required classes will be assigned in OLU and pushed to TTS employees via email.
 * Insider Threat Awareness & Reporting
 * Occupational Health and Safety
 * Plain Language
-* What is Record Management
+* What is Record Management?
 
 ### Affects Designated Employees and other Compliance Tasks
-* OGE Form 450 Financial Disclosure (annual reporting for designated employees only)
 * Telework Training
 
-*NOTE: GSA OLU training material is often not compatible with modern, open-source browsers. We recommend using [VmWare Horizon](/vmware-horizon) and Internet Explorer to ensure successful course completion and documentation.*
+*NOTE: GSA OLU training material sometimes has trouble with certain browsers. We recommend using [VmWare Horizon](/vmware-horizon) and Internet Explorer or, on Mac, Safari, to ensure successful course completion and documentation.*
```

Comment:
I _think_ Chrome is often more successful. This also might be a good time to put together some of the workarounds we've collected in slack (check your popups, flash permissions, etc)

Reply:
I find Chrome has always worked well for me too - I was copying direct recommendations from Marshall on this one.